tflib/publisher/main.tf

```diff
@@ -6,7 +6,7 @@ terraform {
     }
     apko = {
       source  = "chainguard-dev/apko"
-      version = "0.9.2"
```

Comment:
You need to update the digest in the reproducibility check or this will fail postsubmit.